src/types.ts

```diff
@@ -23,6 +23,7 @@ export interface TemplateProps<D extends Document> {
 export interface Template<D extends Document> {
   id: string;
   label: string;
+  type: string;
```

Comment:
the type here is used by developed to create new template, I would suggest to remain it untouched